app/views/instance_user_role_request_mailer/new_role_request.text.erb

```diff
@@ -1,13 +0,0 @@
-<% user = @request.user %>
```

Comment:
From the [create a mailer view](http://guides.rubyonrails.org/action_mailer_basics.html) section, it seems like keeping the text version is best practice?

Reply:
Didn't realise they had some use >_<